### PR TITLE
docs(yard): add file-uploader

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: documentation-site/components/yard/config/file-uploader.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
+import {FileUploader} from 'baseui/file-uploader';
+import {PropTypes} from '../const';
+import {TConfig} from '../types';
+
+const changleHandlers = [
```
Comment (tajo) [inline documentation-site/components/yard/config/file-uploader.ts]: `changeHandlers`?

File: documentation-site/components/yard/config/file-uploader.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      type: PropTypes.Boolean,
+      description: 'Renders component in disabled state.',
+    },
+    errorMessage: {
+      value: '',
+      type: PropTypes.String,
+      description: 'Error message to be displayed.',
+      stateful: true,
+    },
+    ...changleHandlers.reduce((acc, current) => {
```
Comment (tajo) [inline documentation-site/components/yard/config/file-uploader.ts]: I wonder if we should come up with some more robust mechanism to share parts of the documentation. Maybe having `config/common.ts` with all these handlers and then just picking them across other config files. We already do a lot of sharing for input components. Wdyt?

File: documentation-site/components/yard/config/file-uploader.ts (hunk 1 above)
Comment (gergelyke) [reply documentation-site/components/yard/config/file-uploader.ts]: lol, yes :)

File: documentation-site/components/yard/config/file-uploader.ts (hunk 2 above)
Comment (gergelyke) [reply documentation-site/components/yard/config/file-uploader.ts]: that makes sense - I'll do that as part of this PR

**Pull request merged**